src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -16,8 +16,19 @@
 var ReactFiberReconciler = require('ReactFiberReconciler');
 var ReactGenericBatching = require('ReactGenericBatching');
 var emptyObject = require('emptyObject');
+var ReactTypeOfWork = require('ReactTypeOfWork');
```

Comment:
Can you confirm that this works in the npm build? I.e. `grunt build` puts this file in the npm package. I've had trouble in RN when this isn't reachable from within the npm build?